test.js

```diff
@@ -0,0 +1,14 @@
+
+
+document.getElementById('btn-menu').addEventListener('click', toggleMenuPopup)
+const ad = document.getElementById('links')
+const closeBtn = document.querySelector('.close-popup');
+
+function toggleMenuPopup() {
+  ad.classList.toggle('menu-popup');
+}
+
+closeBtn.addEventListener('click', () => {
```

Comment:
- [ ] Kindly add the necessary JS logic for the menu to totally disappear after clicking one of its nav links. See the screenshot below after an attempt to click on the contacts link.
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/225313699-9041f402-44ce-487d-b870-8017271e910f.png)